### PR TITLE
interfaces: fix network-manager plug

### PR DESCRIPTION
**Pull request by mvo5**
Add networkManagerPermanentPlugSecComp that adds socket AF_NETLINK - KOBJECT_UEVENT to unbreak nmcli.

We should squash-merge this and then cherry-pick it for 2.27.

### CODE REVIEW

File: interfaces/builtin/network_manager.go [hunk 1]
```diff
@@ -459,6 +464,11 @@ func (iface *networkManagerInterface) SecCompPermanentSlot(spec *seccomp.Specifi
 	return nil
 }
 
+func (iface *networkManagerInterface) SecCompPermanentPlug(spec *seccomp.Specification, plug *interfaces.Plug) error {
```
Comment (zyga) [inline interfaces/builtin/network_manager.go]: Permanent plug? Why not connected?

**Comment by mvo5** [comment]
We will need to squash merge this to make the 2.27 cherry pick simpler.

**Comment by jdstrand** [comment]
LGTM, +1

**Comment by niemeyer** [comment]
@jdstrand Sending the +1 and the change requests (or even relevant comments) with the actual review feature of GH makes your review tracked both in the summary at the right above, and also in our [review board](/t/review-sprint-3/1880).

**Pull request merged**